1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
+A DC/OS cluster is a set of networked DC/OS nodes with a quorum of master nodes and any number of public and/or private agent nodes.
 
-A Mesos agent node runs a discrete Mesos task on behalf of a framework. It is an agent instance registered with the Mesos master. The synonym of agent node is worker or slave node. See also [private][1] and [public][2] agent nodes.
+### <a name="network"></a>Network
 
-# <a name="bootstrap"></a>Bootstrap node
+DC/OS has two types of networks: infrastructure networks and virtual networks.
 
-The node where a custom DC/OS installation is run. For more information, see the [system requirements](/docs/1.8/administration/installing/custom/system-requirements/#bootstrap-node).
+#### <a name="infrastructure-network"></a>Infrastructure Network
 
-# <a name="cloudtemplate"></a>Cloud template
+An infrastructure network is a physical or virtual network provided by the infrastructure on which DC/OS runs. DC/OS does not manage or control this networking layer, but requires it to exist in order for DC/OS nodes to communicate.
 
-The [cloud templates][3] are optimized to run DC/OS. The templates are JSON-formatted text files that describe the resources and properties.
+#### <a name="virtual-network"></a>Virtual Network
 
-# <a name="mesoscontainerizer"></a>Mesos Containerizer
+A DC/OS virtual network is specifically an overlay network internal to the cluster that connects DC/OS components and containerized tasks running on DC/OS.
 
-The [Mesos Containerizer][4] provides lightweight containerization and resource isolation of executors using Linux-specific functionality such as cgroups and namespaces.
+- The overlay network provided by DC/OS is VXLAN managed by Navstar.
+- Virtual networks must be configured by an administrator before being used by tasks
+- Tasks on DC/OS may opt-in to being placed on a specific virtual network and given a container-specific IP.
+- Virtual networks allow logical subdivision of the tasks running on DC/OS
+- Each task on a virtual network may be configured with optional address groups that virtually isolate communication to tasks on the same network and address group.
 
-# <a name="dockercontainerizer"></a>Docker Containerizer
+### <a name="node"></a>Node
 
-The Docker Containerizer enables launching docker containers using DC/OS.
+A DC/OS node is a virtual or physical machine on which a Mesos agent and/or Mesos master process runs. DC/OS nodes are networked together to form a DC/OS cluster.
 
-# <a name="datacenteroperatingsystem"></a>Datacenter operating system
+#### <a name="master-node"></a>Master Node
 
-A new class of operating system that spans all of the machines in a datacenter or cloud and organizes them to act as one big computer.
+A DC/OS master node is a virtual or physical machine that runs a collection of DC/OS components that work together to manage the rest of the cluster.
 
-# <a name="dcos"></a>DC/OS
+- Each master node contains (among other things) a Mesos master process.
```

Comment:
Should we add a bullet to explain how there must be an odd number of master (e.g. 1, 3, or 5)?